jekyll/_cci2/concepts.md

```diff
@@ -0,0 +1,209 @@
+Each job may contain special steps for caching dependencies from previous jobs to speed up the build.
+
+{% raw %}
+
+```YAML
+version 2
+jobs:
+  build1:
+    docker: # Each job requires specifying an executor
+    # (either docker, macos, or machine), see
+    # circleci.com/docs/2.0/executor-types/ for a comparison
+    # and more examples.
+      - image: circleci/ruby:2.4-node
+      - image: circleci/postgres:9.4.12-alpine
+    steps:
+      - checkout
+      - save_cache: # Caches dependencies with a cache key
+      # template for an environment variable,
+      # see circleci.com/docs/2.0/caching/
+          key: v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+          paths:
+            - ~/circleci-demo-workflows
+
+  build2:
+    docker:
+      - image: circleci/ruby:2.4-node
+      - image: circleci/postgres:9.4.12-alpine
+    steps:
+      - restore_cache: # Restores the cached dependency.
+          key: v1-repo-{{ .Environment.CIRCLE_SHA1 }}       
+```
+
+{% endraw %}
+
+## Workflows
+
+Workflows define a list of jobs and their run order. Within the CI/CD industry, this feature is also referred to as Pipelines. It is possible to run jobs in parallel, sequentially, on a schedule, or with a  manual gate using an approval job.
+
+![workflows illustration]( {{ site.baseurl }}/assets/img/docs/workflow_detail.png)
+
+{% raw %}
+```YAML
+version 2
+jobs:
+  build1:
+    docker:
+      - image: circleci/ruby:2.4-node
+      - image: circleci/postgres:9.4.12-alpine
+    steps:
+      - checkout
+      - save_cache: # Caches dependencies with a cache key
+          key: v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+          paths:
+            - ~/circleci-demo-workflows
+      
+  build2:
+    docker:
+      - image: circleci/ruby:2.4-node
+      - image: circleci/postgres:9.4.12-alpine
+    steps:
+      - restore_cache: # Restores the cached dependency.
+          key: v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+      - run:
+          name: Running tests
+          command: make test
+  build3:
+    docker:
+      - image: circleci/ruby:2.4-node
+      - image: circleci/postgres:9.4.12-alpine
+    steps:
+      - restore_cache: # Restores the cached dependency.
+          key: v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+      - run:
+          name: Precompile assets
+          command: bundle exec rake assets:precompile
+...                          
+workflows:
+  version: 2
+  build_and_test: # name of your workflow
+    jobs:
+      - build1
+      - build2:
+        requires:
+           - build1 # wait for build1 job to complete successfully before starting
+           # see circleci.com/docs/2.0/workflows/ for more examples.
+      - build3:
+        requires:
+           - build1 # wait for build1 job to complete successfully before starting
+           # run build2 and build3 in parallel to save time.
+```
+{% endraw %}
+
+### Workspaces and Artifacts
```

Comment:
Do you want to compare `workspaces` to `caches` or `artifacts`?

Reply:
yes! I've added the comparison table at the end with a link to our blog series on these.